include/boost/fairness/control_block_t.hpp

```diff
@@ -19,32 +18,78 @@
 
 namespace boost::fairness{
 
```

Comment:
Internals of the lib do not require doxygen comments.
If anything control_block definitions now need to be moved in the detail folder.

Reply:
?